Framework/include/QualityControl/CheckerDataProcessor.h

```diff
@@ -35,71 +35,94 @@ namespace checker {
 /// TODO Evaluate whether we should have a dedicated device to store in the database.
 ///
 /// \author Barthélémy von Haller
-class CheckerDataProcessor : public framework::Task {
```

Comment:
There is actually discrepancy between clang-format in O2 and QualityControl (i think QC is closer to Coding Guidelines). Maybe you could mention that in WP3?

Reply:
I will, thank you.